sw/device/sca/lib/sca.c

```diff
@@ -309,7 +309,8 @@ void sca_set_trigger_low(void) {
   OT_DISCARD(dif_gpio_write(&gpio, trigger_bit_index, false));
 }
 
-void sca_call_and_sleep(sca_callee callee, uint32_t sleep_cycles) {
+void sca_call_and_sleep(sca_callee callee, uint32_t sleep_cycles,
+                        bool trigger) {
```

Comment:
nit: maybe "sw_trigger" instead of "trigger" for readability; but not mandatory

Reply:
Good point, I've renamed the parameter.